code/modules/research/research_disk.dm

```diff
@@ -20,3 +20,12 @@
 /obj/item/disk/tech_disk/debug/Initialize()
 	. = ..()
 	stored_research = new /datum/techweb/admin
+
+/obj/item/disk/tech_disk/illegal

```

Comment:
Why should mining get free access to unlocking antag gear?

Reply:
Pooj there is literally a black and red toolbox in the exact same room that unlocks illegal tech.

Infact now that I say that I question why I even added the illegal disk if there's illegal tech right next to it.